packages/components/alert-dialog/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'react'
+
+import { AlertDialog as Root, AlertDialogProps } from './AlertDialog'
+import { AlertDialogAction } from './AlertDialogAction'
+import { AlertDialogBody } from './AlertDialogBody'
+import { AlertDialogCancel } from './AlertDialogCancel'
+import { AlertDialogContent } from './AlertDialogContent'
+import { AlertDialogDescription } from './AlertDialogDescription'
+import { AlertDialogFooter } from './AlertDialogFooter'
+import { AlertDialogHeader } from './AlertDialogHeader'
+import { AlertDialogOverlay } from './AlertDialogOverlay'
+import { AlertDialogPortal } from './AlertDialogPortal'
+import { AlertDialogTitle } from './AlertDialogTitle'
+import { AlertDialogTrigger } from './AlertDialogTrigger'
+
+export * from './AlertDialog'
+export { useAlertDialog } from './AlertDialogContext'
+export { type AlertDialogActionProps } from './AlertDialogAction'
+export { type AlertDialogBodyProps } from './AlertDialogBody'
+export { type AlertDialogCancelProps } from './AlertDialogCancel'
+export { type AlertDialogContentProps } from './AlertDialogContent'
+export { type AlertDialogDescriptionProps } from './AlertDialogDescription'
+export { type AlertDialogFooterProps } from './AlertDialogFooter'
+export { type AlertDialogHeaderProps } from './AlertDialogHeader'
+export { type AlertDialogOverlayProps } from './AlertDialogOverlay'
+export { type AlertDialogPortalProps } from './AlertDialogPortal'
+export { type AlertDialogTitleProps } from './AlertDialogTitle'
+export { type AlertDialogTriggerProps } from './AlertDialogTrigger'
```

Comment:
You might not need this whole block of type imports, as you already export them here: `packages/components/dialog/src/index.ts`

Reply:
IMO one thing is the alert dialog props and another the dialog props and shouldn't be mixed

Reply:
Oh I thought it was exported from the same package, ok